### PR TITLE
Fix grammar (whatsnew_20.rst)

### PR DESCRIPTION
**Pull request by Dreamsorcerer**

### CODE REVIEW

File: doc/build/changelog/whatsnew_20.rst [hunk 1]
```diff
@@ -751,7 +751,7 @@ The new ORM Declarative features introduced above at
 :ref:`whatsnew_20_orm_declarative_typing` introduced the
 new :func:`_orm.mapped_column` construct and illustrated type-centric
 mapping with optional use of :pep:`593` ``Annotated``.  We can take
-the mapping one step further by integrating with with Python
+the mapping one step further by integrating this with Python
```
Comment (CaselIT) [inline doc/build/changelog/whatsnew_20.rst:754]: I'm not a native English speaker, but it sounds better without this. What do you think @zzzeek ?

**Review by CaselIT — COMMENTED** [review]
thanks for noticing

File: doc/build/changelog/whatsnew_20.rst (hunk 1 above)
Comment (zzzeek) [reply doc/build/changelog/whatsnew_20.rst:754]: i think its fine!

**Review by sqla-tester — COMMENTED** [review]
OK, this is **sqla-tester** setting up my work on behalf of **zzzeek** to try to get revision 7488c752d90ce305076ba06b7a4077dc67c0ea66 of this pull request into gerrit so we can run tests and reviews and stuff

**Comment by sqla-tester** [comment]
New Gerrit review created for change 7488c752d90ce305076ba06b7a4077dc67c0ea66: https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4268

**Comment by sqla-tester** [comment]
Gerrit review https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4268 has been **merged**. Congratulations! :)

**Pull request closed without merging**